src/modules/registrypreview/RegistryPreviewUI/MainWindow.xaml

```diff
@@ -41,166 +52,135 @@
+                <AppBarButton
                         x:Name="openButton"
                         x:Uid="OpenButton"
-                        HorizontalAlignment="Left"
-                        Click="OpenButton_Click"
-                        IsTabStop="False">
-                        <AppBarButton.Icon>
-                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xe8e5;"/>
-                        </AppBarButton.Icon>
-                        <AppBarButton.KeyboardAccelerators>
-                            <KeyboardAccelerator Key="O" Modifiers="Control" />
-                        </AppBarButton.KeyboardAccelerators>
-                    </AppBarButton>
-                    <AppBarSeparator />
-                    <AppBarButton
+                        Click="OpenButton_Click">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xe8e5;"/>
+                    </AppBarButton.Icon>
+                    <AppBarButton.KeyboardAccelerators>
+                        <KeyboardAccelerator Key="O" Modifiers="Control" />
+                    </AppBarButton.KeyboardAccelerators>
+                </AppBarButton>
+                <AppBarButton
+                        x:Name="refreshButton"
+                        x:Uid="RefreshButton"
+                        Click="RefreshButton_Click">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xe72c;"/>
+                    </AppBarButton.Icon>
+                    <AppBarButton.KeyboardAccelerators>
+                        <KeyboardAccelerator Key="F5" />
+                    </AppBarButton.KeyboardAccelerators>
+                </AppBarButton>
+                <AppBarSeparator />
+                <AppBarButton
                         x:Name="saveButton"
                         x:Uid="SaveButton"
-                        HorizontalAlignment="Left"
                         Click="SaveButton_Click"
-                        IsEnabled="False"
-                        IsTabStop="False">
-                        <AppBarButton.Icon>
-                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xe74e;"/>
-                        </AppBarButton.Icon>
-                        <AppBarButton.KeyboardAccelerators>
-                            <KeyboardAccelerator Key="S" Modifiers="Control" />
-                        </AppBarButton.KeyboardAccelerators>
-                    </AppBarButton>
-                    <AppBarButton
+                        IsEnabled="False">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xe74e;"/>
+                    </AppBarButton.Icon>
+                    <AppBarButton.KeyboardAccelerators>
+                        <KeyboardAccelerator Key="S" Modifiers="Control" />
+                    </AppBarButton.KeyboardAccelerators>
+                </AppBarButton>
+                <AppBarButton
```

Comment:
Unelated to this PR: could we combine the Save and Save as with a SplitButton? Similiar to what we use in PowerRename. To reduce the amount of visual commands in the command bar.

Reply:
Nope. At least, not in a CommandBar. Maybe if switched to an AppBar control, but to me that's "overkill" :)

Reply:
I'm trying this now, but I'mma ask you to take this over after this pr. I'm getting halfway there, but it would be easier for you.